scripts/bootstrap-ubuntu-1710.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+# This works for Ubuntu 17.10 only.
+
+# system packages
+
+sudo apt-get update
+sudo apt-get upgrade
+
+sudo apt-get install -y gcc make perl
+sudo apt-get install -y \
```

Comment:
I'm a little bit nervous about using the system packages because WT has a custom OS image that I know nothing about. Could we keep this list to a minimum and use the mongodb toolchain for some dependencies?